src/nvim/fileio.c

```diff
@@ -4991,8 +4991,8 @@ buf_check_timestamp (
       }
     } else if (State > NORMAL_BUSY || (State & CMDLINE) || already_warned) {
       if (*mesg2 != NUL) {
-        strncat(tbuf, "; ", tbuf_len);
-        strncat(tbuf, mesg2, tbuf_len);
+        strncat(tbuf, "; ", tbuf_len - 1);
+        strncat(tbuf, mesg2, tbuf_len - 1);
```

Comment:
@jamessan found an oversight: https://github.com/neovim/neovim/pull/5995#discussion_r97244520: `n` in `strncat` is related to source and not destination. Missed this fact when reading documentation, it is rather inconvenient.